src/Extension/Tasks/SecurityCheckTask.yaml

```diff
@@ -0,0 +1,25 @@
+---
+id: 'validation:security:check'
```

Comment:
According to [conventions](https://github.com/spryker-sdk/sdk/blob/master/docs/conventions.md#task) task id should be `validation:php:security-check`